x-pack/platform/packages/shared/kbn-streams-schema/src/models/ingest/base.ts

```diff
@@ -71,10 +73,12 @@ const wiredIngestSchema: z.Schema<WiredIngest> = z.intersection(
 
 const unwiredStreamDefinitionSchemaBase: z.Schema<UnwiredStreamDefinitionBase> = z.object({
   ingest: unwiredIngestSchema,
+  description: z.string(),
```

Comment:
can we move this and above to StreamDefinitionBase?

Reply:
`unwiredStreamDefinitionSchemaBase`(base) is used by the `wiredStreamUpsertRequestSchema` (api), if I move the description out, it will break the edit stream api (no more description as part of the stream upsert request).